README.adoc

```diff
@@ -3216,6 +3216,42 @@ def some_method(&)
 end
 ----
 
+=== Super Forwarding
+
```

Comment:
I’m struggling to understand how this corresponds to the Arguments forwarding section above.

Both that other section and this new one have blind forwarding examples only. There, they suggest using the … syntax, and it is understandable. But only for the blind forwarding case.
But what about cases where only some of the arguments need to be forwarded? Like eg [this one](https://bugs.ruby-lang.org/issues/16253#note-4) with `def m(foo, …)`? Does it even work?

And moving on to the new addition. This all looks reasonable.
My only concerns are below.

Reply:
>I’m struggling to understand how this corresponds to the Arguments forwarding section above.

It fits thematically at least but I see your point about where I placed it. Perhaps it is better suited to come after ["Using `super` with Arguments" ](https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#super-with-args)?

Reply:
My apologies for not expressing my thoughts clearly.
The place is right, I think this new section fits quite well under the arguments forwarding section, and it relates to it.

My concerns were more related to the existing “arguments forwarding” section, as its only “good example” is something I called “blind forwarding”, ie forwarding of all arguments.
This is not exactly always the case, and the existing section is incomplete and may even be confusing due to lack of coverage of those other cases with its only one good example.

Your addition, in the contrary, is detailed, and we happened to discuss a number of edge cases below, so it’s all good.

I’ll keep this comment open for a potential follow-up to improve the “arguments forwarding” section, but not in the scope of this PR.